package.json

```diff
@@ -33,7 +33,24 @@
     "lint:markdown": "./node_modules/.bin/markdownlint $(git ls-files '*.md') -i ./CHANGELOG.md",
     "lint:markdown:fix": "./node_modules/.bin/markdownlint $(git ls-files '*.md') -i ./CHANGELOG.md --fix",
     "reset": "lerna clean -y && rm -rf node_modules && npm i && npm run compile && npm run lint:fix",
-    "update-ts-configs": "node scripts/update-ts-configs.js"
+    "update-ts-configs": "node scripts/update-ts-configs.js",
+
+    "comment_prepare_1": "echo scripts in this section automatically prepare releases. Intended for use by maintainers only.",
+    "comment_prepare_2": "echo experimental preparation scripts only prepare experimental packages",
+    "prepare_release:experimental:patch": "npm run _check:no_changes && npm run _backup:lerna && npm run _lerna:remove_api && npm run _lerna:remove_stable && npm run _lerna:version_patch && npm run _restore:lerna",
+    "prepare_release:experimental:minor": "npm run _check:no_changes && npm run _backup:lerna && npm run _lerna:remove_api && npm run _lerna:remove_stable && npm run _lerna:version_minor && npm run _restore:lerna",
+    "comment_prepare_3": "echo sdk preparation scripts prepare all stable and experimental packages",
+    "prepare_release:sdk:patch": "npm run _check:no_changes && npm run _backup:lerna && npm run _lerna:remove_api && npm run _lerna:version_patch && npm run _restore:lerna",
+    "prepare_release:sdk:minor": "npm run _check:no_changes && npm run _backup:lerna && npm run _lerna:remove_api && npm run _lerna:version_minor && npm run _restore:lerna",
+
+    "comment_internal": "echo scripts below this line are for internal use",
+    "_check:no_changes": "if [ ! -z \"$(git status -uno --porcelain)\" ]; then echo Please ensure all changes are committed; exit 1; fi",
```

Comment:
What was the reason to go with `-uno` instead of `-uall`? :thinking:
It would it be possible that something would make it into a release unintentionally, right? (via untracked files for instance)

Reply:
Ah good catch this is caught over from local testing